docs/pages/User/Configure/discovery.md

```diff
@@ -15,10 +15,10 @@ Liqo simplifies this process by defining three way of discovering a remote clust
 
 <!-- TODO We're in the user guide. I would avoid talking about ForeignCluster CR, if possible. I would simply list the required paramenter, then tell the user how to set them. -->
 
-In `ForeignCluster` CR there are required parameters to connect to a foreign cluster
+We need some parameters to contact and to connect to a remote cluster:
 
 * `ClusterID`: cluster's unique identifier, it is created by Liqo during first run
-* `LiqoNamespace`: namespace where Liqo components are deployed and they expect to find resources required to work
+* `LiqoNamespace`: namespace where Liqo components are deployed and they expect to find resources required to work. If you installed Liqo with the [provided script](../../gettingstarted/install/) the namespace should be `liqo`
 * `IP` and `Port` of the Kubernetes API Server
 
 <!-- TODO As discussed in the weekly call on 18/08, not clear why we need to specify ClusterID, instead of allowing the system to discover that parameter automatically -->
```

Comment:
@frisso @palexster  I'm working on a proposal regarding this point, to move not strictly required parameters to a "cluster manifest" somewhere

This "cluster manifest" in the future could also contain soma other things like features/services that a cluster can support and want to show to the world before peering

If we have time, next week we can discuss it

Reply:
👍